velox/connectors/hive/storage_adapters/gcs/GCSFileSystem.h

```diff
@@ -20,50 +20,70 @@
 
 namespace facebook::velox::filesystems {
 
-// Implementation of GCS filesystem and file interface.
-// We provide a registration method for read and write files so the appropriate
-// type of file can be constructed based on a filename. See the
-// (register|generate)ReadFile and (register|generate)WriteFile functions.
+/// Implementation of GCS filesystem and file interface.
```

Comment:
nit. single line comments should start with // ( not /// ), applies to this entire file.

Reply:
Deepak had let me know in a previous comment that /// are for documentation comments in headers.

From the CODING style:
For comments on headers for classes, functions, methods and member variables (the ones to be exposed as documentation), use /// (three slashes) at the front of each line.